diagnostics/convective_transition_diag/convecTransCriticalCollapse_util.py

```diff
@@ -586,7 +587,7 @@ def convecTransCriticalCollapse_plot(argsv1,argsv2,argsv3,argsv4,argsv5,argsv6):
         elif (BULK_TROPOSPHERIC_TEMPERATURE_MEASURE==2):
             footnote_str+='$\widehat{q_{sat}}$ (1000-200hPa Column-integrated Saturation Specific Humidity) as the bulk tropospheric temperature measure'
         footnote_str+='\n$w_c$ estimated by fitting (dashed) the average precip. pickup curves for the 3 most probable temperature bins'
-        ax1.text(s=footnote_str, x=0, y=-0.02, transform=fig_obs_wc.transFigure, ha='left', va='top', fontsize=12)
+        #ax1.text(s=footnote_str, x=0, y=-0.02, transform=fig_obs_wc.transFigure, ha='left', va='top', fontsize=12)
```

Comment:
Is there a need to keep the old line around?

Reply:
I would like to uncomment this line, but the newer matplotlib version doesn't like this line. Haven't figure it out. Keep it for now.